frontend/src/metabase/collections/containers/CollectionSidebar.jsx

```diff
@@ -46,24 +46,31 @@ class CollectionSidebar extends React.Component {
   state = {
     openCollections: [],
   };
+
+  componentDidMount() {
+    // an array to store the ancestors
+    const { collectionId, collections, loading } = this.props;
+    if (!loading) {
+      const ancestors = getParentPath(collections, Number(collectionId)) || [];
+      this.setState({ openCollections: ancestors });
+    }
+  }
+
   onOpen = id => {
     this.setState({ openCollections: this.state.openCollections.concat(id) });
   };
+
   onClose = id => {
     this.setState({
       openCollections: this.state.openCollections.filter(c => {
         return c !== id;
       }),
     });
   };
-  componentDidMount() {
-    // an array to store the ancestors
-    const { collectionId, collections, loading } = this.props;
-    if (!loading) {
-      const ancestors = getParentPath(collections, Number(collectionId)) || [];
-      this.setState({ openCollections: ancestors });
-    }
-  }
+
+  // TODO Should we update the API to filter archived collections?
```

Comment:
A personal preference on my part but I'd prefer we avoid adding theoretical TODOs since they are often quickly forgotten about.